spec/hbw/features/tasks/task_list_with_filter_bp_mock.yml

```diff
@@ -0,0 +1,375 @@
+get:
+  user:
+    - params: "email=c.johnson@example.com"
+      response:
+        data:
+          - id:         'demo'
+            firstName:  'Demo'
+            lastName:   'Demo'
+            url:        'http://bpm.example.com'
+            email:      'c.johnson@example.com'
+            pictureUrl: null
+        total: 1
+        start: 0
+        sort: 'id'
+        order: 'asc'
+        size: 1
+
+  # process definitions
+  process-definition:
+    - params: "latestVersion=true&startableBy=c.johnson@example.com"
+      response:
+        - &default_process-definition
+          id: "testProcess:469:222221"
+          key: "support_request"
+          version: 469
+          name: "Test name"
+          description: null
+          tenantId: null
+          deploymentId: "222221"
+          resource: "testProcess.bpmn"
+          diagram: null
+          category: "http://bpmn.io/schema/bpmn"
+          suspended: false
+          versionTag: null
+          historyTimeToLive: null
+        - <<: *default_process-definition
+          id: "testProcess:469:222222"
+          deploymentId: "222222"
+
+  # form definitions
+  task/111111/deployed-form: &form_with_everything
```

Comment:
why do you need the form with everything?

Reply:
Removed